### PR TITLE
Added RID restrictions mapping with SDP examples

### LINKED ISSUE #97: Additional coverage of MANE/SFU issues
There may be a few things that aren't currently covered: 

* Negotiation of an alternative forwarding header extension (e.g. frame-marking), instead of the Dependency Descriptor.  This is viable for SFU forwarding only for simple streams (e.g. temporal scalability only).

* Coverage of potential security issues relating to the Dependency Descriptor.  These issues can arise in situations where the SFU is not trusted.

### PR DESCRIPTION
**Pull request by dsychev**
- added interpretation for RID restrictions when applied to AV1
- fixed example for 3 simulcast streams. Every stream to have its own payload type
- extended SDP example to signal resolution with RIDs
- added SDP example with dependant temporal layers

fixes #97

### CODE REVIEW

File: av1-rtp-spec.md [hunk 1]
```diff
@@ -305,10 +305,21 @@ The RTP payload defined in this specification supports two distinct modes for tr
...
+When simulcast encodings are transported on a single RTP stream, **depend** restriction from [I-D.ietf-avtext-rid] could be used to signal dependencies between scalable layers. This simulcast transport mode is possible since [AV1] enables multiple simulcast encodings to be provided within a single bitstream. However, in this mode, RTCP feedback cannot be provided for each simulcast encoding, but only for the aggregate, since only a single SSRC is used. This mode of simulcast transport MAY be supported by SFUs.
 
+### 6.1.1 RID restrictions mapping for AV1
 
-### 6.1.1 Example
+RID specification declares the set of codec-agnostic restrictions for media streams. All the restrictions are optional and are subject to negotiation based on the SDP Offer/Answer rules described in Section 6 if [I-D.ietf-mmusic-rid]. When restrictions applied to AV1 codec, they must have following interpretation:
```
Comment (DanilChapovalov) [inline av1-rtp-spec.md:312]: s/if [I-D.ietf-mmusic-rid]/in [I-D.ietf-mmusic-rid]/

Comment (dsychev) [reply av1-rtp-spec.md]: fixed

**Comment by aboba** [comment]
draft-ietf-avtext-rid does not mention depend restrictions, this is defined in draft-ietf-mmusic-rid. Unfortunately, depend is used "to identify other streams that the stream depends on" and in this case there is only one stream.

**Comment by aboba** [comment]
In the single RTP stream simulcast example, RIDs are used.  The implication is that the RID header extension would be used to differentiate between the simulcast encodings. The problem is that RID stands for "RTP Stream Identifier" not "Encoding Identifier".  So it's not clear that it can be used that way.

**Comment by aboba** [comment]
In the SDP examples, the max-width and max-height values are in the wrong order.  It should be: 

a=rid:f recv max-width=1920;max-height=1080
a=rid:h recv max-width=1280;max-height=720
a=rid:q recv max-width=640;max-height=480

**Comment by aboba** [comment]
This PR has now been rebased and split into two PRs: https://github.com/AOMediaCodec/av1-rtp-spec/pull/123 and https://github.com/AOMediaCodec/av1-rtp-spec/pull/124

**Pull request closed without merging**